web/inc/helpers.php

```diff
@@ -50,6 +51,7 @@ function exit_code_to_http_code(int $exit_code, int $default = 400): int {
 		case E_UNSUSPENDED:
 		case E_FORBIDEN:
 		case E_FORBIDDEN:
+		case E_API_DISABLED:
 			return 401;
```

Comment:
alignment of return should be checked.

Reply:
<img width="610" alt="Screenshot 2023-06-21 at 14 46 46" src="https://github.com/hestiacp/hestiacp/assets/9754650/d4cb6521-2c0a-442e-8f49-4d5b9deaf53e">

Looks fine and consistent to me...